README.md

```diff
@@ -62,8 +64,23 @@ docker run -d \
     --dns=127.0.0.1 --dns=1.1.1.1 \
     pihole/pihole:latest
 
-echo -n "Your password for https://${IP}/admin/ is "
-docker logs pihole 2> /dev/null | grep 'password:'
+
+printf 'Starting up pihole container '
```

Comment:
I don't want this much arbitrary bash on the first page of the readme.  I don't mind hiding it in the script.  At this point I feel like instead of mirroring the contents of docker_run.sh we just link to it.